Khan/wonder-blocks
# Add Checkbox to Storybook

## Summary:
Add Checkbox Storybook documentation.
All of the documentation is taken directly from styleguidist.

Issue: https://khanacademy.atlassian.net/browse/WB-1289

<img width="1063" alt="Screen Shot 2022-04-29 at 7 00 18 PM" src="https://user-images.githubusercontent.com/13231763/166086244-8193dd64-f4a5-42ab-a055-d0a38eb0bfbf.png">
<img width="1112" alt="Screen Shot 2022-04-29 at 7 00 22 PM" src="https://user-images.githubusercontent.com/13231763/166086248-29c75890-85d6-470c-a59d-386790730f75.png">
<img width="1137" alt="Screen Shot 2022-04-29 at 6 58 42 PM" src="https://user-images.githubusercontent.com/13231763/166086222-53bcd86b-83a6-4e11-9c7f-c38bdd130e19.png">
<img width="1073" alt="Screen Shot 2022-04-29 at 6 58 46 PM" src="https://user-images.githubusercontent.com/13231763/166086224-3b03e554-3978-4240-9c30-74f8f7b249b5.png">


## Test plan:
- Go to http://localhost:6061/?path=/docs/form-checkbox--checkbox-default
- Compare with the docs in https://wonder-blocks.netlify.app/#checkbox
- Make sure that the controls work for the default story.
